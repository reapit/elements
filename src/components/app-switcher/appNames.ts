export type AppName =
  | 'reapitPM'
  | 'reapitSales'
  | 'reapitLetting'
  | 'reapitForms'
  | 'reapitWebsites'
  | 'reapitProposals'
  | 'keyWhere'
  | 'autoResponder'

type AppInfo = {
  name: string
  description: string
}

export const apps: Record<AppName, AppInfo> = {
  reapitPM: { name: 'Reapit PM', description: 'Property Management' },
  reapitSales: { name: 'Reapit Sales', description: 'Real Estate CRM' },
  reapitLetting: { name: 'Reapit Lettings', description: 'Lettings Management' },
  reapitForms: { name: 'Reapit Forms', description: 'Documents & Digital Signing' },
  reapitWebsites: { name: 'Reapit Websites', description: 'Bespoke Website Design' },
  reapitProposals: { name: 'Reapit Proposals', description: 'Interactive Digital Proposals' },
  keyWhere: { name: 'KeyWhere', description: 'Real-time Key Management' },
  autoResponder: { name: 'Auto Responder', description: 'Automated Email Marketing' },
}
