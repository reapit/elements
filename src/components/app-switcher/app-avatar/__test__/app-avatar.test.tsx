import { render } from '@testing-library/react'
import AppAvatar from '../app-avatar'
import { AppName, apps } from '../../appNames'

const testCases = Object.keys(apps) as AppName[]

test.each(testCases)('renders %s icon when `hasAccess` is true', (appName) => {
  const { asFragment } = render(<AppAvatar appName={appName} hasAccess={true} />)
  expect(asFragment()).toMatchSnapshot()
})

test.each(testCases)('renders %s icon when `hasAccess` is false', (appName) => {
  const { asFragment } = render(<AppAvatar appName={appName} hasAccess={false} />)
  expect(asFragment()).toMatchSnapshot()
})
