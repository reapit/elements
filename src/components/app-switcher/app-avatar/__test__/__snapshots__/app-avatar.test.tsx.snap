// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`renders agentBox icon when \`hasAccess\` is false 1`] = `
<DocumentFragment>
  <svg
    fill="none"
    font-size="40"
    height="1em"
    viewBox="0 0 40 40"
    width="1em"
    xmlns="http://www.w3.org/2000/svg"
  >
    <path
      d="M0 8C0 3.58172 3.58172 0 8 0H32C36.4183 0 40 3.58172 40 8V32C40 36.4183 36.4183 40 32 40H8C3.58172 40 0 36.4183 0 32V8Z"
      fill="url(#paint0_linear_681_1707)"
    />
    <g
      clip-path="url(#clip0_681_1707)"
    >
      <path
        d="M10.5934 6.28003V9.98303H25.3403V29.1294H29.3891V6.28003H10.5934Z"
        fill="white"
      />
      <path
        d="M20.9394 33.72V13.9604L10.5934 9.98303L10.5934 29.7461L20.9394 33.72Z"
        fill="white"
      />
    </g>
    <defs>
      <lineargradient
        gradientUnits="userSpaceOnUse"
        id="paint0_linear_681_1707"
        x1="20.7019"
        x2="20.7019"
        y1="0.350877"
        y2="31.2281"
      >
        <stop
          offset="0.075"
          stop-color="#798DA1"
        />
        <stop
          offset="0.575"
          stop-color="#607890"
        />
      </lineargradient>
      <clippath
        id="clip0_681_1707"
      >
        <rect
          fill="white"
          height="28"
          transform="translate(6 6)"
          width="28"
        />
      </clippath>
    </defs>
  </svg>
</DocumentFragment>
`;

exports[`renders agentBox icon when \`hasAccess\` is true 1`] = `
<DocumentFragment>
  <svg
    fill="none"
    font-size="40"
    height="1em"
    viewBox="0 0 40 40"
    width="1em"
    xmlns="http://www.w3.org/2000/svg"
  >
    <path
      d="M0 8C0 3.58172 3.58172 0 8 0H32C36.4183 0 40 3.58172 40 8V32C40 36.4183 36.4183 40 32 40H8C3.58172 40 0 36.4183 0 32V8Z"
      fill="url(#paint0_linear_681_72)"
    />
    <g
      clip-path="url(#clip0_681_72)"
    >
      <path
        d="M10.5934 6.28003V9.98303H25.3403V29.1294H29.3891V6.28003H10.5934Z"
        fill="white"
      />
      <path
        d="M20.9394 33.72V13.9604L10.5934 9.98303L10.5934 29.7461L20.9394 33.72Z"
        fill="white"
      />
    </g>
    <defs>
      <lineargradient
        gradientUnits="userSpaceOnUse"
        id="paint0_linear_681_72"
        x1="20.7019"
        x2="20.7019"
        y1="0.350877"
        y2="31.2281"
      >
        <stop
          stop-color="#5573FD"
        />
        <stop
          offset="0.575"
          stop-color="#4E56EA"
        />
      </lineargradient>
      <clippath
        id="clip0_681_72"
      >
        <rect
          fill="white"
          height="28"
          transform="translate(6 6)"
          width="28"
        />
      </clippath>
    </defs>
  </svg>
</DocumentFragment>
`;

exports[`renders autoResponder icon when \`hasAccess\` is false 1`] = `
<DocumentFragment>
  <svg
    fill="none"
    font-size="40"
    height="1em"
    viewBox="0 0 40 40"
    width="1em"
    xmlns="http://www.w3.org/2000/svg"
  >
    <path
      d="M0 8C0 3.58172 3.58172 0 8 0H32C36.4183 0 40 3.58172 40 8V32C40 36.4183 36.4183 40 32 40H8C3.58172 40 0 36.4183 0 32V8Z"
      fill="#F2F4F6"
    />
    <path
      d="M17.0094 20.6776V18.3905L9.1665 25.2124L17.0094 31.9583V29.7696C23.6642 29.6883 29.7498 26.3678 29.7498 23.0145V16.3454C26.5921 20.6396 18.3383 20.6966 17.0094 20.6776Z"
      fill="#607890"
    />
    <path
      d="M29.4371 13.6183C28.8558 8.30581 20.3087 7.25079 17.0093 7.04163V15.7093C21.4846 15.9008 24.1145 17.3034 25.0753 17.7401C25.847 17.3742 28.5524 15.9532 29.4371 13.6183Z"
      fill="#607890"
    />
  </svg>
</DocumentFragment>
`;

exports[`renders autoResponder icon when \`hasAccess\` is true 1`] = `
<DocumentFragment>
  <svg
    fill="none"
    font-size="40"
    height="1em"
    viewBox="0 0 40 40"
    width="1em"
    xmlns="http://www.w3.org/2000/svg"
  >
    <path
      d="M0 8C0 3.58172 3.58172 0 8 0H32C36.4183 0 40 3.58172 40 8V32C40 36.4183 36.4183 40 32 40H8C3.58172 40 0 36.4183 0 32V8Z"
      fill="#ECF3FF"
    />
    <path
      d="M17.0094 20.6776V18.3905L9.1665 25.2124L17.0094 31.9583V29.7696C23.6642 29.6883 29.7498 26.3678 29.7498 23.0145V16.3454C26.5921 20.6396 18.3383 20.6966 17.0094 20.6776Z"
      fill="#4E56EA"
    />
    <path
      d="M29.4371 13.6183C28.8558 8.30581 20.3087 7.25079 17.0093 7.04163V15.7093C21.4846 15.9008 24.1145 17.3034 25.0753 17.7401C25.847 17.3742 28.5524 15.9532 29.4371 13.6183Z"
      fill="#4E56EA"
    />
  </svg>
</DocumentFragment>
`;

exports[`renders consoleCloud icon when \`hasAccess\` is false 1`] = `
<DocumentFragment>
  <svg
    fill="none"
    font-size="40"
    height="1em"
    viewBox="0 0 40 40"
    width="1em"
    xmlns="http://www.w3.org/2000/svg"
  >
    <path
      d="M0 8C0 3.58172 3.58172 0 8 0H32C36.4183 0 40 3.58172 40 8V32C40 36.4183 36.4183 40 32 40H8C3.58172 40 0 36.4183 0 32V8Z"
      fill="url(#paint0_linear_681_3376)"
    />
    <g
      clip-path="url(#clip0_681_3376)"
    >
      <path
        d="M8.11975 13.1456V26.8544L19.9911 33.7081L31.8631 26.8544V19.2783L30.2111 18.3242L27.4636 16.738L26.4045 16.1262L26.4031 20.8561L26.4017 25.586H25.1788H14.802H13.5784V24.3631V17.3757V16.1521L16.7844 14.3013L18.9313 13.0623L19.9904 12.4505L23.1971 14.2887L25.3433 15.5186L26.4045 16.1262V14.9033V9.99423L19.9911 6.29193L8.11975 13.1456Z"
        fill="white"
      />
    </g>
    <defs>
      <lineargradient
        gradientUnits="userSpaceOnUse"
        id="paint0_linear_681_3376"
        x1="20.7019"
        x2="20.7019"
        y1="0.350877"
        y2="31.2281"
      >
        <stop
          offset="0.075"
          stop-color="#798DA1"
        />
        <stop
          offset="0.575"
          stop-color="#607890"
        />
      </lineargradient>
      <clippath
        id="clip0_681_3376"
      >
        <rect
          fill="white"
          height="28"
          transform="translate(6 6)"
          width="28"
        />
      </clippath>
    </defs>
  </svg>
</DocumentFragment>
`;

exports[`renders consoleCloud icon when \`hasAccess\` is true 1`] = `
<DocumentFragment>
  <svg
    fill="none"
    font-size="40"
    height="1em"
    viewBox="0 0 40 40"
    width="1em"
    xmlns="http://www.w3.org/2000/svg"
  >
    <path
      d="M0 8C0 3.58172 3.58172 0 8 0H32C36.4183 0 40 3.58172 40 8V32C40 36.4183 36.4183 40 32 40H8C3.58172 40 0 36.4183 0 32V8Z"
      fill="url(#paint0_linear_203_7948)"
    />
    <g
      clip-path="url(#clip0_203_7948)"
    >
      <path
        d="M8.11963 13.1456V26.8544L19.9909 33.7081L31.8629 26.8544V19.2783L30.2109 18.3242L27.4634 16.738L26.4043 16.1262L26.4029 20.8561L26.4015 25.586H25.1786H14.8018H13.5782V24.3631V17.3757V16.1521L16.7842 14.3013L18.9311 13.0623L19.9902 12.4505L23.1969 14.2887L25.3431 15.5186L26.4043 16.1262V14.9033V9.99423L19.9909 6.29193L8.11963 13.1456Z"
        fill="white"
      />
    </g>
    <defs>
      <lineargradient
        gradientUnits="userSpaceOnUse"
        id="paint0_linear_203_7948"
        x1="20.7019"
        x2="20.7019"
        y1="0.350877"
        y2="31.2281"
      >
        <stop
          stop-color="#5573FD"
        />
        <stop
          offset="0.575"
          stop-color="#4E56EA"
        />
      </lineargradient>
      <clippath
        id="clip0_203_7948"
      >
        <rect
          fill="white"
          height="28"
          transform="translate(6 6)"
          width="28"
        />
      </clippath>
    </defs>
  </svg>
</DocumentFragment>
`;

exports[`renders ireWeb icon when \`hasAccess\` is false 1`] = `
<DocumentFragment>
  <svg
    fill="none"
    font-size="40"
    height="1em"
    viewBox="0 0 40 40"
    width="1em"
    xmlns="http://www.w3.org/2000/svg"
  >
    <path
      d="M0 8C0 3.58172 3.58172 0 8 0H32C36.4183 0 40 3.58172 40 8V32C40 36.4183 36.4183 40 32 40H8C3.58172 40 0 36.4183 0 32V8Z"
      fill="url(#paint0_linear_681_188)"
    />
    <path
      d="M32.2415 6.15323C12.4847 6.15323 9.05886 25.5243 7.74146 33.4273H9.37526C9.81276 30.5279 12.7773 19.0619 19.9915 13.6663C19.9915 13.6663 14.5042 20.4619 13.59 28.3271C16.9276 27.6915 27.0762 25.042 29.2301 15.7348L25.4886 15.1419C26.806 13.8245 30.271 10.1054 32.2408 6.15393L32.2415 6.15323Z"
      fill="white"
    />
    <defs>
      <lineargradient
        gradientUnits="userSpaceOnUse"
        id="paint0_linear_681_188"
        x1="20.7019"
        x2="20.7019"
        y1="0.350877"
        y2="31.2281"
      >
        <stop
          offset="0.075"
          stop-color="#798DA1"
        />
        <stop
          offset="0.575"
          stop-color="#607890"
        />
      </lineargradient>
    </defs>
  </svg>
</DocumentFragment>
`;

exports[`renders ireWeb icon when \`hasAccess\` is true 1`] = `
<DocumentFragment>
  <svg
    fill="none"
    font-size="40"
    height="1em"
    viewBox="0 0 40 40"
    width="1em"
    xmlns="http://www.w3.org/2000/svg"
  >
    <path
      d="M0 8C0 3.58172 3.58172 0 8 0H32C36.4183 0 40 3.58172 40 8V32C40 36.4183 36.4183 40 32 40H8C3.58172 40 0 36.4183 0 32V8Z"
      fill="url(#paint0_linear_681_1909)"
    />
    <path
      d="M32.2415 6.15326C12.4847 6.15326 9.05886 25.5244 7.74146 33.4274H9.37526C9.81276 30.528 12.7773 19.062 19.9915 13.6664C19.9915 13.6664 14.5042 20.462 13.59 28.3272C16.9276 27.6916 27.0762 25.0421 29.2301 15.7349L25.4886 15.142C26.806 13.8246 30.271 10.1055 32.2408 6.15396L32.2415 6.15326Z"
      fill="white"
    />
    <defs>
      <lineargradient
        gradientUnits="userSpaceOnUse"
        id="paint0_linear_681_1909"
        x1="20.7019"
        x2="20.7019"
        y1="0.350877"
        y2="31.2281"
      >
        <stop
          stop-color="#5573FD"
        />
        <stop
          offset="0.575"
          stop-color="#4E56EA"
        />
      </lineargradient>
    </defs>
  </svg>
</DocumentFragment>
`;

exports[`renders keywhere icon when \`hasAccess\` is false 1`] = `
<DocumentFragment>
  <svg
    fill="none"
    font-size="40"
    height="1em"
    viewBox="0 0 40 40"
    width="1em"
    xmlns="http://www.w3.org/2000/svg"
  >
    <path
      d="M0 8C0 3.58172 3.58172 0 8 0H32C36.4183 0 40 3.58172 40 8V32C40 36.4183 36.4183 40 32 40H8C3.58172 40 0 36.4183 0 32V8Z"
      fill="#F2F4F6"
    />
    <g
      clip-path="url(#clip0_681_199)"
    >
      <path
        d="M30.9102 23.9576L26.3854 19.4328L30.682 15.1362C32.4565 13.3617 32.4565 10.4546 30.682 8.68007C28.9075 6.90557 26.0004 6.90557 24.2259 8.68007L16.8864 16.1246L16.8717 16.1519V11.7748C16.8717 9.26317 14.8179 7.20935 12.3063 7.20935C9.79471 7.20935 7.74091 9.26317 7.74091 11.7748V27.0501C7.74091 28.3052 8.25331 29.4463 9.08141 30.2737L9.07231 30.2898C10.8468 32.0643 13.7539 32.0643 15.5284 30.2898L19.958 25.8601L24.4541 30.4129C26.2286 32.1874 29.1357 32.1874 30.9102 30.4129C32.6847 28.6384 32.6847 25.7313 30.9102 23.9569V23.9576ZM12.3063 7.89817C14.4231 7.89817 16.1402 9.61528 16.1402 11.7321C16.1402 13.8489 14.4231 15.566 12.3063 15.566C10.1895 15.566 8.47241 13.8489 8.47241 11.7321C8.47241 9.61528 10.1895 7.89817 12.3063 7.89817ZM12.3336 30.8848C10.2308 30.8848 8.52491 29.181 8.52491 27.0761C8.52491 24.9712 10.2287 23.2674 12.3336 23.2674C14.4385 23.2674 16.1423 24.9712 16.1423 27.0761C16.1423 29.181 14.4385 30.8848 12.3336 30.8848ZM23.6106 11.564C23.7611 9.45215 25.5951 7.86245 27.707 8.01295C29.8189 8.16345 31.4086 9.99745 31.2581 12.1094C31.1076 14.2206 29.2736 15.8109 27.1617 15.6604C25.0498 15.5099 23.4601 13.6759 23.6106 11.564ZM27.9128 31.1221C25.747 31.2432 23.8906 29.5863 23.7695 27.4184C23.6484 25.2526 25.3053 23.3962 27.4732 23.2751C29.6411 23.154 31.4954 24.8109 31.6165 26.9788C31.7376 29.1467 30.0807 31.001 27.9128 31.1221Z"
        fill="#607890"
      />
    </g>
    <defs>
      <clippath
        id="clip0_681_199"
      >
        <rect
          fill="white"
          height="28"
          transform="translate(6 6)"
          width="28"
        />
      </clippath>
    </defs>
  </svg>
</DocumentFragment>
`;

exports[`renders keywhere icon when \`hasAccess\` is true 1`] = `
<DocumentFragment>
  <svg
    fill="none"
    font-size="40"
    height="1em"
    viewBox="0 0 40 40"
    width="1em"
    xmlns="http://www.w3.org/2000/svg"
  >
    <path
      d="M0 8C0 3.58172 3.58172 0 8 0H32C36.4183 0 40 3.58172 40 8V32C40 36.4183 36.4183 40 32 40H8C3.58172 40 0 36.4183 0 32V8Z"
      fill="#ECF3FF"
    />
    <g
      clip-path="url(#clip0_681_8218)"
    >
      <path
        d="M30.9103 23.9576L26.3855 19.4328L30.6821 15.1362C32.4566 13.3617 32.4566 10.4546 30.6821 8.68007C28.9076 6.90557 26.0005 6.90557 24.226 8.68007L16.8865 16.1246L16.8718 16.1519V11.7748C16.8718 9.26317 14.818 7.20935 12.3064 7.20935C9.79477 7.20935 7.74097 9.26317 7.74097 11.7748V27.0501C7.74097 28.3052 8.25337 29.4463 9.08147 30.2737L9.07237 30.2898C10.8469 32.0643 13.754 32.0643 15.5285 30.2898L19.9581 25.8601L24.4542 30.4129C26.2287 32.1874 29.1358 32.1874 30.9103 30.4129C32.6848 28.6384 32.6848 25.7313 30.9103 23.9569V23.9576ZM12.3064 7.89817C14.4232 7.89817 16.1403 9.61528 16.1403 11.7321C16.1403 13.8489 14.4232 15.566 12.3064 15.566C10.1896 15.566 8.47247 13.8489 8.47247 11.7321C8.47247 9.61528 10.1896 7.89817 12.3064 7.89817ZM12.3337 30.8848C10.2309 30.8848 8.52497 29.181 8.52497 27.0761C8.52497 24.9712 10.2288 23.2674 12.3337 23.2674C14.4386 23.2674 16.1424 24.9712 16.1424 27.0761C16.1424 29.181 14.4386 30.8848 12.3337 30.8848ZM23.6107 11.564C23.7612 9.45215 25.5952 7.86245 27.7071 8.01295C29.819 8.16345 31.4087 9.99745 31.2582 12.1094C31.1077 14.2206 29.2737 15.8109 27.1618 15.6604C25.0499 15.5099 23.4602 13.6759 23.6107 11.564ZM27.9129 31.1221C25.7471 31.2432 23.8907 29.5863 23.7696 27.4184C23.6485 25.2526 25.3054 23.3962 27.4733 23.2751C29.6412 23.154 31.4955 24.8109 31.6166 26.9788C31.7377 29.1467 30.0808 31.001 27.9129 31.1221Z"
        fill="#4E56EA"
      />
    </g>
    <defs>
      <clippath
        id="clip0_681_8218"
      >
        <rect
          fill="white"
          height="28"
          transform="translate(6 6)"
          width="28"
        />
      </clippath>
    </defs>
  </svg>
</DocumentFragment>
`;

exports[`renders reapitForms icon when \`hasAccess\` is false 1`] = `
<DocumentFragment>
  <svg
    fill="none"
    font-size="40"
    height="1em"
    viewBox="0 0 40 40"
    width="1em"
    xmlns="http://www.w3.org/2000/svg"
  >
    <path
      d="M0 8C0 3.58172 3.58172 0 8 0H32C36.4183 0 40 3.58172 40 8V32C40 36.4183 36.4183 40 32 40H8C3.58172 40 0 36.4183 0 32V8Z"
      fill="#F2F4F6"
    />
    <g
      clip-path="url(#clip0_681_1435)"
    >
      <path
        d="M32.6 8.0007L6 20.32L15.1896 22.6741L26.748 13.9458L19.0263 24.5683L29.0328 33.2882L32.5993 8L32.6 8.0007Z"
        fill="#607890"
      />
    </g>
    <defs>
      <clippath
        id="clip0_681_1435"
      >
        <rect
          fill="white"
          height="28"
          transform="translate(6 6)"
          width="28"
        />
      </clippath>
    </defs>
  </svg>
</DocumentFragment>
`;

exports[`renders reapitForms icon when \`hasAccess\` is true 1`] = `
<DocumentFragment>
  <svg
    fill="none"
    font-size="40"
    height="1em"
    viewBox="0 0 40 40"
    width="1em"
    xmlns="http://www.w3.org/2000/svg"
  >
    <path
      d="M0 8C0 3.58172 3.58172 0 8 0H32C36.4183 0 40 3.58172 40 8V32C40 36.4183 36.4183 40 32 40H8C3.58172 40 0 36.4183 0 32V8Z"
      fill="#ECF3FF"
    />
    <g
      clip-path="url(#clip0_681_134)"
    >
      <path
        d="M32.6 8.0007L6 20.32L15.1896 22.6741L26.748 13.9458L19.0263 24.5683L29.0328 33.2882L32.5993 8L32.6 8.0007Z"
        fill="#4E56EA"
      />
    </g>
    <defs>
      <clippath
        id="clip0_681_134"
      >
        <rect
          fill="white"
          height="28"
          transform="translate(6 6)"
          width="28"
        />
      </clippath>
    </defs>
  </svg>
</DocumentFragment>
`;

exports[`renders reapitProposals icon when \`hasAccess\` is false 1`] = `
<DocumentFragment>
  <svg
    fill="none"
    font-size="40"
    height="1em"
    viewBox="0 0 40 40"
    width="1em"
    xmlns="http://www.w3.org/2000/svg"
  >
    <path
      d="M0 8C0 3.58172 3.58172 0 8 0H32C36.4183 0 40 3.58172 40 8V32C40 36.4183 36.4183 40 32 40H8C3.58172 40 0 36.4183 0 32V8Z"
      fill="#F2F4F6"
    />
    <path
      d="M16.037 11.6481C14.255 10.1176 11.8381 9.13276 9.1665 9.01983V26.198C12.1231 26.3116 14.7987 27.3962 16.7671 29.0869C17.7355 29.9188 18.5333 30.8978 19.1098 31.9805V17.9933C19.1098 15.5233 17.9325 13.2758 16.037 11.6481Z"
      fill="#607890"
    />
    <path
      d="M30.8332 9.01929C28.1615 9.13288 25.744 10.1177 23.9627 11.6476C22.0672 13.2759 20.8899 15.5221 20.8899 17.9921V31.9793C21.4664 30.8966 22.2641 29.9176 23.2326 29.0857C25.201 27.395 27.8766 26.3104 30.8332 26.1968V9.01929Z"
      fill="#607890"
    />
  </svg>
</DocumentFragment>
`;

exports[`renders reapitProposals icon when \`hasAccess\` is true 1`] = `
<DocumentFragment>
  <svg
    fill="none"
    font-size="40"
    height="1em"
    viewBox="0 0 40 40"
    width="1em"
    xmlns="http://www.w3.org/2000/svg"
  >
    <path
      d="M0 8C0 3.58172 3.58172 0 8 0H32C36.4183 0 40 3.58172 40 8V32C40 36.4183 36.4183 40 32 40H8C3.58172 40 0 36.4183 0 32V8Z"
      fill="#ECF3FF"
    />
    <path
      d="M16.037 11.6481C14.255 10.1176 11.8381 9.13276 9.1665 9.01983V26.198C12.1231 26.3116 14.7987 27.3962 16.7671 29.0869C17.7355 29.9188 18.5333 30.8978 19.1098 31.9805V17.9933C19.1098 15.5233 17.9325 13.2758 16.037 11.6481Z"
      fill="#4E56EA"
    />
    <path
      d="M30.8332 9.01929C28.1615 9.13288 25.744 10.1177 23.9627 11.6476C22.0672 13.2759 20.8899 15.5221 20.8899 17.9921V31.9793C21.4664 30.8966 22.2641 29.9176 23.2326 29.0857C25.201 27.395 27.8766 26.3104 30.8332 26.1968V9.01929Z"
      fill="#4E56EA"
    />
  </svg>
</DocumentFragment>
`;

exports[`renders reapitWebsites icon when \`hasAccess\` is false 1`] = `
<DocumentFragment>
  <svg
    fill="none"
    font-size="40"
    height="1em"
    viewBox="0 0 40 40"
    width="1em"
    xmlns="http://www.w3.org/2000/svg"
  >
    <path
      d="M0 8C0 3.58172 3.58172 0 8 0H32C36.4183 0 40 3.58172 40 8V32C40 36.4183 36.4183 40 32 40H8C3.58172 40 0 36.4183 0 32V8Z"
      fill="#F2F4F6"
    />
    <path
      d="M17.7135 16.581L20.7708 19.6383C23.123 21.9905 25.6102 19.8952 26.5642 18.3731C27.1256 17.4774 27.6314 16.6207 28.0022 15.7878C28.8701 13.8375 29.0032 12.2876 28.3577 11.6422C27.7122 10.9967 26.163 11.1297 24.212 11.9977C22.1511 12.9153 19.8658 14.5306 17.7135 16.581Z"
      fill="#607890"
    />
    <path
      d="M26.0683 9.74059C21.7181 7.16326 16.0815 7.59226 12.159 11.0269L16.58 15.448C19.9604 12.2166 23.4367 10.1861 26.0683 9.74059Z"
      fill="#607890"
    />
    <path
      d="M11.027 12.1585C7.59239 16.0811 7.16339 21.7177 9.74069 26.068C10.1856 23.4364 12.216 19.96 15.4481 16.5796L11.027 12.1585Z"
      fill="#607890"
    />
    <path
      d="M20.0147 27.3895L20.0167 27.3876C17.8002 28.9685 15.6797 29.9642 13.9326 30.2595C18.2828 32.8368 23.9194 32.4078 27.8419 28.9732L26.1061 27.2373C23.5387 24.6699 21.74 26.2369 20.0147 27.3895Z"
      fill="#607890"
    />
    <path
      d="M16.2954 27.7642C18.8164 26.3799 22.5715 23.7026 19.1461 20.2778L16.5814 17.7131C14.5311 19.8661 12.9157 22.1514 11.9988 24.2124C11.1309 26.1627 10.9978 27.7126 11.6433 28.3581C12.2888 29.0036 13.8379 28.8705 15.789 28.0025C15.9565 27.9277 16.1253 27.8476 16.2954 27.7642Z"
      fill="#607890"
    />
    <path
      d="M30.2598 13.9315C29.956 15.7289 28.91 17.9203 27.2496 20.2056C27.1132 20.3817 26.9742 20.5658 26.8385 20.7571C25.8507 22.1487 25.0602 23.9276 27.1503 26.0184L28.9735 27.8416C32.4075 23.919 32.8371 18.2817 30.2598 13.9315Z"
      fill="#607890"
    />
  </svg>
</DocumentFragment>
`;

exports[`renders reapitWebsites icon when \`hasAccess\` is true 1`] = `
<DocumentFragment>
  <svg
    fill="none"
    font-size="40"
    height="1em"
    viewBox="0 0 40 40"
    width="1em"
    xmlns="http://www.w3.org/2000/svg"
  >
    <path
      d="M0 8C0 3.58172 3.58172 0 8 0H32C36.4183 0 40 3.58172 40 8V32C40 36.4183 36.4183 40 32 40H8C3.58172 40 0 36.4183 0 32V8Z"
      fill="#ECF3FF"
    />
    <path
      d="M17.7135 16.581L20.7708 19.6383C23.123 21.9905 25.6102 19.8952 26.5642 18.3731C27.1256 17.4774 27.6314 16.6207 28.0022 15.7878C28.8701 13.8375 29.0032 12.2876 28.3577 11.6422C27.7122 10.9967 26.163 11.1297 24.212 11.9977C22.1511 12.9153 19.8658 14.5306 17.7135 16.581Z"
      fill="#4E56EA"
    />
    <path
      d="M26.0683 9.74059C21.7181 7.16326 16.0815 7.59226 12.159 11.0269L16.58 15.448C19.9604 12.2166 23.4367 10.1861 26.0683 9.74059Z"
      fill="#4E56EA"
    />
    <path
      d="M11.027 12.1585C7.59239 16.0811 7.16339 21.7177 9.74069 26.068C10.1856 23.4364 12.216 19.96 15.4481 16.5796L11.027 12.1585Z"
      fill="#4E56EA"
    />
    <path
      d="M20.0147 27.3895L20.0167 27.3876C17.8002 28.9685 15.6797 29.9642 13.9326 30.2595C18.2828 32.8368 23.9194 32.4078 27.8419 28.9732L26.1061 27.2373C23.5387 24.6699 21.74 26.2369 20.0147 27.3895Z"
      fill="#4E56EA"
    />
    <path
      d="M16.2954 27.7642C18.8164 26.3799 22.5715 23.7026 19.1461 20.2778L16.5814 17.7131C14.5311 19.8661 12.9157 22.1514 11.9988 24.2124C11.1309 26.1627 10.9978 27.7126 11.6433 28.3581C12.2888 29.0036 13.8379 28.8705 15.789 28.0025C15.9565 27.9277 16.1253 27.8476 16.2954 27.7642Z"
      fill="#4E56EA"
    />
    <path
      d="M30.2598 13.9315C29.956 15.7289 28.91 17.9203 27.2496 20.2056C27.1132 20.3817 26.9742 20.5658 26.8385 20.7571C25.8507 22.1487 25.0602 23.9276 27.1503 26.0184L28.9735 27.8416C32.4075 23.919 32.8371 18.2817 30.2598 13.9315Z"
      fill="#4E56EA"
    />
  </svg>
</DocumentFragment>
`;
